eslint.config.mjs

```diff
@@ -0,0 +1,44 @@
+import pluginJs from "@eslint/js";
+import stylisticJs from "@stylistic/eslint-plugin-js";
+import importPlugin from "eslint-plugin-import";
+import jsdoc from "eslint-plugin-jsdoc";
+import globals from "globals";
+
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+    {
```

Comment:
I suggest adding these paths to be ignored by ESLint:

```suggestion
export default [
    {
        ignores: [
            "env/",
            "dist/",
            "node_modules/",
            "bagitobjecttransfer/static/",
        ],
    },
    {
```

I had trouble getting this to work, but it's working the way I have it writtne. I thought `ignores` would go inside each configuration, but it's supposed to go inside it's own configuration like I have it above. See:

- https://eslint.org/docs/latest/use/configure/ignore#ignoring-directories
- https://github.com/eslint/eslint/discussions/18304#discussioncomment-9069706